src/roles/nuh-deploy/tasks/main.yml

```diff
@@ -128,6 +128,20 @@
         with_items:
           - "{{ external_interfaces }}"
 
+      - name: Copy proxy certificates over to secondary nuh
```

Comment:
Please add a resolved issue to RELEASE_NOTES.md

Reply:
I thought about adding one but this was introduced in current dev. so I did not add one

Reply:
Ok, don't need to add it if you are just fixing a problem in current dev support.